docs/core/tutorials/debugging-with-visual-studio.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Debug your C# Hello World application with Visual Studio 2017
-description: Learn how to debug a Hello World app written in C# with Visual Studio 2017.
+title: Debug your Hello World application with Visual Studio 2017
```

Comment:
I was thinking of adding some more keywords to this title to help SEO, such as
"Debug your C# or Visual Basic Hello World .NET Core application with Visual Studio 2017"